README.md

```diff
@@ -27,6 +27,9 @@
 - **Tech Stack:** We are thinking about Typescript, Go or C++, as the team has people familiar with this but we will consider the comfort of the contributors.
 - **Distribution**: How we distribute the tool depends on the tech stack we choose. We will revisit this when we decide the tech stack.
 
+## Note about AI usage
+   The use of Artificial Intelligence is discouraged either in your contributions or proposal preparation, as it doesn’t help us learn at the early stage in the open source community. The contributors or applicants of GSoC can include a paragraph in their proposal, that it has been written without the help of AI.
```

Comment:
nit: indentation fix
```suggestion

The use of Artificial Intelligence is discouraged either in your contributions or proposal preparation, as it doesn’t help us learn at the early stage in the open source community. The contributors or applicants of GSoC can include a paragraph in their proposal, that it has been written without the help of AI.
```

Reply:
@RaisinTen Is there any reason for this change? I wanted it to look like a paragraph. Do you follow any style guide?

Reply:
Ah, it doesn't look like a paragraph right now either, see - https://github.com/postman-open-technologies/openapi-componentizer/blob/541630d0c3feaba7792796946cce61a075eec49c/README.md#note-about-ai-usage. I suggested this because it's consistent with how the rest of the text is written.